asv_bench/benchmarks/algorithms.py

```diff
@@ -112,5 +119,25 @@ def time_series_timedeltas(self, df):
     def time_series_dates(self, df):
         hashing.hash_pandas_object(df['dates'])
 
+    def time_series_multiindexes(self, df):
+        hashing.hash_pandas_object(df.index)
+
+
+class Quantile(object):
+    params = [[0, 0.5, 1], ['linear', 'nearest', 'lower', 'higher',
+                            'midpoint']]
+    param_names = ['quantile', 'interpolation']
+
+    def setup(self, quantile, interpolation):
+        N = 10**5
+        self.int_idx = pd.Series(np.arange(N).repeat(5))
```

Comment:
Can you just parametrize the different types here? Seems more straightforward than assigning these as is during setup

Reply:
Modified entire file to parametrize types